.github/workflows/ci.yaml

```
@@ -60,6 +60,7 @@ jobs:
   verify-test:
     name: "Verify & Test"
     runs-on: ubuntu-latest
+    environment: dev
```

nit:
```suggestion
    environment: development
```
to be aligned with another environment names.